alinka/docx/__init__.py

```diff
@@ -6,11 +6,12 @@
 from .utils import convert_raw_documents_data
 
 
-def generate_documents(record_id: str) -> None:
+def generate_documents(record_id: str, destination_path: str) -> None:
     list_of_documents_to_generate = [DocumentsTypes.ZARZADZANIE, DocumentsTypes.ZAWIADOMIENIE, DocumentsTypes.PROTOKOL]
     raw_documents_data = get_decision_data_from_db(record_id)
     document_data = convert_raw_documents_data(raw_documents_data)
-    destination_path = settings.DOCUMENTS_PATH
+    if not destination_path:
+        destination_path = settings.DOCUMENTS_PATH
```

Comment:
Ok, I see, what wll happen, when this is not set - why not then just omit these 2 lines and ☝🏿 ?